Base/usr/share/man/man5/af.md

```diff
@@ -1,18 +1,49 @@
 ## Name
 
-af - Application File format
+af - Application File format (.af)
 
 ## Synopsis
 
 The Application Files define System Menu entries and launcher file types / protocols.
 
 ## Description
 
-.af files are human-readable and are a subset of the INI-format, have no easily detectable filemagic. These files define System Menu entries and launcher file types / protocols.
+Application files are a subset of the INI format.
+They have no easily detectable filemagic and contain application information (App group):
 
-They are stored in [`/res/apps`](../../../../res/apps).
+| key           | description                      |
+|---------------|----------------------------------|
+| Name          | name                             |
+| Executable    | executable path                  |
+| Category      | category (optional)              |
+| Description   | description (optional)           |
+| IconPath      | application icon path (optional) |
+| RunInTerminal | run in terminal flag (optional)  |
 
-## See Also
+and launcher information (Launcher group, optional):
 
+| key       | description                           |
+|-----------|---------------------------------------|
+| FileTypes | supported file types separated by ',' |
+| Protocols | protocols separated by ','            |
+
+-------------------------------------------------------
```

Comment:
We don't use separator line in any man pages AFAIK.

```suggestion
```

Reply:
Some do have it, like the Help start page, but I'd absolutely avoid it.